.github/workflows/unit_tests.yml

```diff
@@ -33,7 +33,7 @@ jobs:
         run: |
           python -m pip install --upgrade pip
           pip install pytest
-          pip install -e .[test]
+          pip install -e .[schemas,test]
```

Comment:
This will be switched to `all` once I'm sure that I haven't broken anything with the optional install of server

Reply:
Or perhaps this could just be kept as the long list of explicit extras, excluding `server`....